spec.html

```diff
@@ -0,0 +1,652 @@
+        1. If _kind_ is ~base~, then
+          1. Let _thisArgument_ be ? OrdinaryCreateFromConstructor(_newTarget_, *"%Object.prototype%"*).
+        1. Let _calleeContext_ be PrepareForOrdinaryCall(_F_, _newTarget_).
+        1. Assert: _calleeContext_ is now the running execution context.
+        1. If _kind_ is ~base~, then
+          1. Perform OrdinaryCallBindThis(_F_, _calleeContext_, _thisArgument_).
+          1. Let _initializeResult_ be InitializeInstanceElements(_thisArgument_, _F_).
+          1. If _initializeResult_ is an abrupt completion, then
+            1. Remove _calleeContext_ from the execution context stack and restore _callerContext_ as the running execution context.
+            1. Return Completion(_initializeResult_).
+        1. Let _constructorEnv_ be the LexicalEnvironment of _calleeContext_.
+        1. Let _result_ be OrdinaryCallEvaluateBody(_F_, _argumentsList_).
+        1. Remove _calleeContext_ from the execution context stack and restore _callerContext_ as the running execution context.
+        1. If _result_.[[Type]] is ~return~, then
+          1. If Type(_result_.[[Value]]) is Object, return NormalCompletion(_result_.[[Value]]).
+          1. <del>If _kind_ is ~base~, return NormalCompletion(_thisArgument_).</del>
+          1. <ins>If _kind_ is ~base~, then</ins>
+            1. <ins>If _F_.[[Brand]] is not ~empty~, then append _F_.[[Brand]] to _thisArgument_.[[Brands]]</ins>
+            1. <ins>Return NormalCompletion(_thisArgument_).</ins>
+          1. If _result_.[[Value]] is not *undefined*, throw a *TypeError* exception.
+        1. Else, ReturnIfAbrupt(_result_).
+        1. <ins>Let _currentThis_ be ? _constructorEnv_.GetThisBinding().</ins>
+        1. <ins>If _F_.[[Brand]] is not ~empty~, then append _F_.[[Brand]] to _currentThis_.[[Brands]]</ins>
+        1. <del>Return ? _constructorEnv_.GetThisBinding().</del>
+        1. Return _currentThis_.
+      </emu-alg>
+    </emu-clause>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-ecmascript-language-expressions">
+  <h1>ECMAScript Language: Expressions</h1>
+
+  <emu-clause id="sec-left-hand-side-expressions">
+    <h1>Left-Hand-Side Expressions</h1>
+    <h2>Syntax</h2>
+    <emu-grammar type="definition">
+      CallExpression[Yield, Await] :
+        CoverCallExpressionAndAsyncArrowHead[?Yield, ?Await] #callcover
+        SuperCall[?Yield, ?Await]
+        ImportCall[?Yield, ?Await]
+        <ins>BrandCheckCall[?Yield, ?Await]</ins>
+        CallExpression[?Yield, ?Await] Arguments[?Yield, ?Await]
+        CallExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await] `]`
+        CallExpression[?Yield, ?Await] `.` IdentifierName
+        CallExpression[?Yield, ?Await] TemplateLiteral[?Yield, ?Await, +Tagged]
+        CallExpression[?Yield, ?Await] `.` PrivateIdentifier
+
+      <ins>BrandCheckCall[Yield, Await] :</ins>
+        `class` `.` `hasInstance` `(` AssignmentExpression[+In, ?Yield, ?Await] `)`
```

Comment:
```suggestion
        `class` `.` `hasInstance` `(` AssignmentExpression[+In, ?Yield, ?Await] `,`? `)`
```
Should we allow trailing comma here? I would expect consistency between call expression and brand check call.

Reply:
I believe we should allow that. I remember `import()` also have supported that.